### DISCUSSION TITLE
Compiled Release?

### DISCUSSION THREAD

**Discussion by Tiddalick** [KajiTetsushi/HomeworldSDL#44] [Q&A]
Hi there,

Is there a compiled release binary for this project?
This seems the most active of the Homeworld open source efforts and I have an urge to play this great game once more.
I am not much of a programmer, so looking through the build instructions left me a little cold.
I would much rather play this then the remastered version!

**Comment by KajiTetsushi** [comment]
@Tiddalick Thanks for stopping by! Playing this instead of the Remastered edition I'm pretty sure is why we're all here.

I'll be very very sincere here: I tried to "inherit" this code with zero C experience, so I'm lacking in many areas. Release strategy is one of them. I'd like to get to this quickly, though. No point in working on it if nobody can get around to use it.

I'm maining on macOS Mojave right now which immediately compiles the game into debug mode (I think?). But I imagine that there are more Windows and Linux people than macOS so you have a very good point.

**Reply by Tiddalick** [reply]
@KajiTetsushi Well I see there is a document that shows how to compile the code, so even if I don't have a huge amount of experience I can have a go at doing this when I get a minute (windows at least).
I have seen some projects that seems to have an automatic build cycle, but I am not aware of how this is done.

Still, it is just good that people like yourself are continuing to work on the original code. Perhaps there will be a greater resurgence of interest when Homeworld 3 is closer to reality.

I will have a fiddle and let you know if I manage to make anything that runs!
